utils/rpms/daos.spec

```diff
@@ -2,10 +2,11 @@
 
 %global spdk_version 19.04.1
 %global cart_version 4.7.0
```

Comment:
should this not be 4.8 now?

Reply:
I would think it should be.  I guess nobody did update this before RC1 was made.

But the RC1 release was not made with the correct tag in any case.  Seems we need to redo it to get this change, the cart update and the correct tag in place.

Reply:
I should clarify, that it's not a hard requirement.  It just means that DAOS would accept a cart-4.7.x if it were made available during installation.  But since RPM aims to install the highest version RPMs that meet versioning restrictions, if 4.8.0 is available, it will be used.

We will just have to be careful to make sure (only) cart 4.8.0 is included in the release package and therefore available during installation.